sdk/tables/azure-data-tables/azure/data/tables/_table_batch.py

```diff
@@ -234,40 +235,37 @@ def update(self, entity: EntityType, mode: Union[str, UpdateMode] = UpdateMode.M
                 :caption: Creating and adding an entity to a Table
         """
         self._verify_partition_key(entity)
-        temp = entity.copy()  # type: ignore
```

Comment:
Why was this changed?
The reason this is here is to prevent mutation of the customers dictionary - it may not be necessary, but have you confirm this?

Reply:
We do "copy" inside _add_entity_properties(), so it's not necessary to "copy" before calling that function.